worlds/sc2/options.py

```diff
@@ -944,6 +962,23 @@ def get_option_value(world: 'SC2World', name: str) -> Union[int,  FrozenSet]:
     return player_option.value
 
 
+def get_enabled_races(world: 'SC2World') -> Set[SC2Race]:
+    selection = get_option_value(world, 'selected_races')
+    if selection == SelectRaces.option_all:
+        return set(SC2Race)
+    enabled = set()
+    # if bit 0, enable terran
+    if selection & 1:
+        enabled.add(SC2Race.TERRAN)
+    # if bit 1, enable zerg
+    if selection & (1 << 1):
+        enabled.add(SC2Race.ZERG)
+    # if bit 2, enable protoss
+    if selection & (1 << 2):
+        enabled.add(SC2Race.PROTOSS)
+    return enabled
```

Comment:
Recommend making a new intflag or at least giving these magic numbers some names. Not urgent to complete in this PR, I guess, as mission filtering and related options are likely to see some work after Salz merges his stuff.

Reply:
I also recommend factoring in mission exclusions here. If this feels to early to do that, then maybe leave a todo or note when we should handle it.

Reply:
It feels weird to make a new intflag for something we already have an intEnum for (SC2Race in mission_tables), and literally all I'm doing here is parsing an option that we already needed to collapse down for brevity's sake. 

....I guess I could do something crazy like make SelectRace.option_terran return MissionFlag.Terran or something?